### PR TITLE
EN6-77 Improved error handling.

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/aps/system/services/resource/ResourcesServiceUniTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                .thenReturn(TestHelper.createGroups());
+
+        resourcesService.setAuthorizationManager(authorizationManager);
+        resourcesService.setFileAllowedExtensions(Arrays.stream(new String[] { "pdf","txt" }).collect(Collectors.toList()));
+        resourcesService.setImageAllowedExtensions(Arrays.stream(new String[] { "jpeg","png" }).collect(Collectors.toList()));
+    }
+
+    @Test(expected = ValidationGenericException.class)
+    public void testInvalidGroup() {
+        resourcesService.validateGroup(null, "invalidGroup");
```
Comment (joewhite101) [inline plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/aps/system/services/resource/ResourcesServiceUniTest.java]: Seems like assertions are missing on these

Comment (ffleandro) [reply plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/aps/system/services/resource/ResourcesServiceUniTest.java]: The assertion `assertThrows` is only available in JUnit 5 (we're currently using JUnit 4).
The assertion is made using the `expected` param for expecting Exceptions.

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by joewhite101** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Pull request closed without merging**